### PR TITLE
Feature/245 confirm attendance

### LINKED ISSUE #248: Create waiver component
Create a tick box component for the liability waiver in the confirmation page. See Figma for mockups.

### PR DESCRIPTION
**Pull request by pierreTklein**
### Figma URL

https://www.figma.com/file/TU0ycyxLagQ8xio2GhfDKqMj/Registration-system-desktop?node-id=638%3A7

### Card pops up when you have been accepted

screen shot 2018-12-24 at 4 53 12 pm

### Clicking on it brings you to this page

screen shot 2018-12-24 at 4 53 25 pm

### Validation: you must accept the liability

screen shot 2018-12-24 at 4 53 37 pm

### Redirect to home page when successful, removal of card. 

screen shot 2018-12-24 at 4 52 56 pm

## Check the following:

- [ ] User cannot access page unless they are confirmed
- [ ] Card is added once user has accepted
- [ ] User must accept waiver
- [ ] Database is updated
- [ ] Card is removed once user has confirmed
- [ ] No regression

### CODE REVIEW

File: src/ConfirmAttendance/ConfirmAttendance.tsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    return (
+      <Flex
+        justifyContent={'center'}
+        alignItems={'center'}
+        flexDirection={'column'}
+      >
+        <MaxWidthBox width={1}>
+          <H1 textAlign={'center'}>Confirm your Attendance</H1>
+          <Paragraph fontSize={'16px'}>
+            We are excited to offer you a spot at McHacks 6 taking place on
```
Comment (loreina) [inline src/ConfirmAttendance/ConfirmAttendance.tsx]: `We are excited to offer you a spot at McHacks 6, taking place on February 2-3, 2019 at Theatre St. James in the Old Port of Montreal.`

File: src/ConfirmAttendance/ConfirmAttendance.tsx [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+            February 2-3rd 2019 at the Theatre St. James in the Old Port of
+            Montreal.
+          </Paragraph>
+          <Paragraph fontSize={'16px'}>
+            Please confirm your attendance before January 15th.
```
Comment (loreina) [inline src/ConfirmAttendance/ConfirmAttendance.tsx]: January 15, 2019

File: src/ConfirmAttendance/ConfirmAttendance.tsx [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+          </Paragraph>
+        </MaxWidthBox>
+        <MaxWidthBox width={1}>
+          <H2 color={theme.colors.greyDark}>Libaility and Photo Release</H2>
```
Comment (loreina) [inline src/ConfirmAttendance/ConfirmAttendance.tsx]: Liability*

**Comment by pierreTklein** [comment]
Yeah it's technically centered, it's just that `Theatre` is too long to fit on the edge. The page looks pretty clunky, imo. I'm game to receive re-designs.

**Comment by krubenok** [comment]
yeah I'm down to have design look at it. I was mistaken on the closing date for applications so I just threw something together hella hasty to get it done.

**Comment by pierreTklein** [comment]
Should we leave this open until further notice then?

**Comment by krubenok** [comment]
> Should we leave this open until further notice then?

For a day or two yeah. I think even with applications closing later than I thought we might do a round of acceptances before they close.

**Comment by pierreTklein** [comment]
Lmk

**Comment by pierreTklein** [comment]
@loreina can I merge this ?

**Comment by pierreTklein** [comment]
@krubenok we need https://mchacks.ca/liability

**Comment by krubenok** [comment]
@loreina waiting on the PR approval for the liability page on the McHacks site. 
https://github.com/hackmcgill/mchacks6/pull/135

**Pull request merged**